src/parser.js

```diff
@@ -88,6 +88,17 @@ function mapKey(node) {
   }
 }
 
+function isKeyValueMatch(key, value) {
+  if (!n.Identifier.check(value)) {
+    return false;
+  }
+
+  return (
+    (n.Literal.check(key) && key.value === value.name) ||
+    (n.Identifier.check(key) && key.name === value.name)
```

Comment:
That doesn't work because a `Literal` key has a `value`, while an `Identifier` key has a `name`.